### ISSUE TITLE
Unintended(?) answer - from edge 'has part' combined with element

### ISSUE THREAD

**Issue by Genomewide** [NCATSTranslator/Feedback#591]
This looks like an unintended relationship.

https://ui.transltr.io/main/results?l=Diabetes%20Mellitus&i=MONDO:0005015&t=0&q=4fe4e2b4-9761-443c-b535-71d97797d8f7
![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/c095d965-007f-42e9-96f9-a965f2b6f278)

I think I saw this also with Bethyl Myopathy.  There were 50+ food answers that 'had part' -> Nitrogen and a drug had Nitrogen so the food treated the disease.  (Also, the node 99.9% Nitrogen also 'has part' -> Nitrogen and also may treat the disease.

**Comment by Genomewide** [comment]
Another example:

![image](https://github.com/NCATSTranslator/Feedback/assets/55069176/329999ee-c688-4165-bc47-935336c9467c)

**Comment by gglusman** [comment]
Seems like more examples of the faulty 'transitive' reasoning. Added the label.

**Comment by cbizon** [comment]
FWIW, I don't think that transitivity is inherently a problem.  The rules that we derive in ARAGORN are based on statistical calculations: links that match these patterns are associated more frequently than you would expect with the predicted edge.  They are not logical deductions, and they are not meant to be.  The question is - how certain to we need to be? is the score we provide commensurate with the (low) level of certainty that this may provide?   

Now, that said, it seems as though there are ways to add some tighter filters.  Things like going through Nitrogen are probably bad smells.

**Comment by gglusman** [comment]
I agree that transitivity is not _inherently_ a problem. The label is about faulty assumptions of transitivity being appropriate.
See the examples I put [elsewhere](https://github.com/NCATSTranslator/Feedback/issues/468#issuecomment-1674978797).

**Comment by cbizon** [comment]
Right, I appreciate that.  But it's inherently a statistical argument so while recognizing the crummy examples is important in the sense that we might have a way to make things better, there's not an argument here (IMO) that we should remove this kind of rule.

**Comment by gglusman** [comment]
Agree that we don't need to entirely remove transitivity as a rule, but there may be need for a curation effort to annotate, for each type of predicate, whether it is always, sometimes, or never appropriate to treat it as transitive. And for the 'sometimes' (which might be most of them...) it might be worth encoding somehow when they can be treated as transitive.
Not saying anything of this is easy. If such curation cannot be done, it might be worth weakening the confidence in any reasoning that uses transitivity.

**Comment by sierra-moxon** [comment]
@cbizon - is there any action to take here, or do we disagree about the solution and need more discussion?
